tests/remeshing/parallel-explicit/helper.hpp

```diff
@@ -215,6 +215,8 @@ inline void runResetInput(testing::TestContext &context)
           .initialize()
           .write({1.01, 1.02})
           .advance()
+          .resetMesh()
+          .setVertices({2.0, y, 3.0, y})
```

Comment:
To also make the goal of this PR a bit clearer: Was it intended in the original design of this test that the non-primary ranks of `"A"` do not call `resetMesh` or was it an oversight?